examples/line-liff-v1/bottender.config.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  channels: {
+    messenger: {
+      enabled: false,
+      path: '/webhooks/messenger',
```

Comment:
We can keep config simple by removing non-line config too. See:

https://github.com/Yoctol/bottender/blob/master/examples/line-hello-world/bottender.config.js